utils.py

```diff
@@ -58,12 +58,10 @@ def createWorker(worker_name_prefix, worker_id, worker_type, dockerfile, jobs=5,
 
     if volumes is None:
         volumes=['/srv/buildbot/ccache:/mnt/ccache', '/srv/buildbot/packages:/mnt/packages', '/mnt/autofs/master_packages/:/packages']
-    # Set master FQDN - for VPN machines it should be 100.64.100.1
-    fqdn = 'buildbot.mariadb.org'
-    if worker_name_prefix.startswith('intel') or worker_name_prefix.startswith('bg') or worker_name_prefix.startswith('amd'):
-        fqdn = '100.64.100.1'
-    if worker_name_prefix.startswith('ppc64le-rhel'):
-        fqdn = '10.103.203.6'
+    # Set master FQDN - default to wireguard interface
+    fqdn = '100.64.100.1'
+    if worker_name_prefix.startswith('aarch64-bbw[1-4]') or worker_name_prefix.startswith('ppc64le-rhel'):
```

Comment:
This doesn't work as far as I know. Startswith doesn't support regex. Are you certain that it works?

Reply:
Also I know remember that for ppc64 there were various DNS issues, especially after a restart

Reply:
I'll test the DNS pb.

Reply:
> This doesn't work as far as I know. Startswith doesn't support regex. Are you certain that it works?

What about `re.match`